internal/functions/deploy/testdata/modules/imports.ts

```diff
@@ -0,0 +1,90 @@
+import {
```

Comment:
@sweatybridge another scenario we might have to consider is dynamic imports referencing files in the same directory as entrypoint.

Currently this code works when deployed, because we include all source files in a function directory. Parsing dynamic imports would be hard, if the paths are not staticly analyzable (eg: `await import(getFilename())`).

```
await import("./foo.ts")
Deno.serve(() => new Response("ok"))
```

I think in the release we can call it out as a breaking change and suggest:
1. Change local dynamic imports to hoisted imports.
2. Add those files to `staticPatterns` in config.yaml

Reply:
Yup, good shout. But is this case supported by `edge-runtime bundle` command? `getFilename()` could come from a user request for example.

In either case, I plan to keep `--use-docker` as the default flag for the time being.

Reply:
Currently, it's implicitly supported because we bundle all source files within a function's directory.